core/arch/arm/plat-sam/conf.mk

```diff
@@ -45,16 +58,16 @@ $(call force,CFG_CORE_DYN_SHM,n)
 
 CFG_MMAP_REGIONS ?= 24
 
-CFG_SHMEM_START  ?= 0x21000000
+CFG_SHMEM_START  ?= $(shell echo $$(($(CFG_TZDRAM_START) + 0x1000000)))
```

Comment:
Doesn't `CFG_SHMEM_START  ?= ($(CFG_TZDRAM_START) + 0x1000000)` work?
If it does, please use that pattern instead, here and below.

Reply:
Yes, this pattern works. Updated and added the review tags. Thank you.